### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by buhtz**
- added link to Launchpad Translation section
 - removed "Newsfeed" section because it isn't used currently
 - added TOC in known problems section
 - minor modifications in content
 - tried to improved formatting and layout

Please carefully review my English. Thanks.

**EDIT**:
- typo in `./configure` script "usage"
- removed outdated "configure usage" output from README.md
- improved wording (based on an correction tool 😄 )

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,76 +29,57 @@ You only need to specify 3 things:
...
 
-If you use `rsync >= 3.2.4` and `backintime <= 1.3.2` there is a workaround. Add `--old-args` in [_Expert Options_ / _Additional options to rsync_](https://backintime.readthedocs.io/en/latest/settings.html#expert-options).
-
-Note that some GNU/Linux distributions (e.g. Manjaro) using a workaround with environment variable `RSYNC_OLD_ARGS` in there distro-specific packages for _Back In Time_. In that case you may not see any problems.
+If you use `rsync >= 3.2.4` and `backintime <= 1.3.2` there is a workaround. Add `--old-args` in [_Expert Options_ / _Additional options to rsync_](https://backintime.readthedocs.io/en/latest/settings.html#expert-options). Note that some GNU/Linux distributions (e.g. Manjaro) using a workaround with environment variable `RSYNC_OLD_ARGS` in there distro-specific packages for _Back In Time_. In that case you may not see any problems.
```
Comment (aryoda) [inline README.md:55]: "in there distro-specific" should be "in their..."

File: README.md [hunk 2]
```diff
@@ -223,39 +203,33 @@ to be prepared to build an updated version of the package:
...
 ### From sources
 
-To build and install from the source code
-- do a `git clone https://github.com/bit-team/backintime.git` on your computer
-- install the required build and run-time dependencies
-- then build and install with `make` as described below.
-
-The dependencies are described for Ubuntu here.
-If you use another Linux distribution please install the corresponding packages.
+The dependencies based on Ubuntu. Please [open an Issue](https://github.com/bit-team/backintime/issues/new/choose)
```
Comment (aryoda) [inline README.md:206]: Missing "are": "The dependencies are based on Ubuntu"

File: README.md [hunk 3]
```diff
@@ -290,37 +267,27 @@ If you use another Linux distribution please install the corresponding packages.
...
-Note: The first value is default.
-
-See also `common/configure --help` and `qt/configure --help`
-
-
-## NewsFeed
-
-Back In Time has an RSS feed
-https://feeds.launchpad.net/backintime/announcements.atom
+You can use these optional arguments to `./configure` for creating a Makefile.
```
Comment (aryoda) [inline README.md:272]: delete "these" since the list of options is missing now and must be fetched via cmd line

File: README.md [hunk 4]
```diff
@@ -290,37 +267,27 @@ If you use another Linux distribution please install the corresponding packages.
...
-https://mail.python.org/mailman3/lists/bit-dev.python.org/
-
-There is a developer documentation on https://backintime-dev.readthedocs.org
-It's not complete yet but I'm working on it. If you'd like to contribute
-please add docstrings following the
-[Google style guide](https://sphinxcontrib-napoleon.readthedocs.org/en/latest/example_google.html)
-and add unit-tests for new methods in common. To run unit-test locally you can
-run `cd common && ./configure && make && make test`
+### Resources
+ - Mailing list [bit-dev](https://mail.python.org/mailman3/lists/bit-dev.python.org/) for development related topics
```
Comment (aryoda) [inline README.md:277]: I would extend the link name to [Mailing list `bit-dev`]

File: README.md [hunk 5]
```diff
@@ -290,37 +267,27 @@ If you use another Linux distribution please install the corresponding packages.
...
+ - [Source code documentation for developers](https://backintime-dev.readthedocs.org)
+ - [Translations](https://translations.launchpad.net/backintime) are done on a separate plattform
+### Guidelines & Rules
+The maintaining team will welcome all types of contributions. No contribution will be rejected
```
Comment (aryoda) [inline README.md:281]: maintaining -> maintenance ?

File: README.md [hunk 6]
```diff
@@ -290,37 +267,27 @@ If you use another Linux distribution please install the corresponding packages.
...
+just because it doesn't fit to our quality standards, guidelines or rules. Every contribution
+is reviewed and if needed will be improved together with the maintainers.
+
+If you can and know how it will improve the process and lower the burden of maintain if the
```
Comment (aryoda) [inline README.md:285]: Many details, I would shorten this e.g. into:

"Please take the following best practices into account if possible (to reduce the work load of the maintainers):"

**Review by aryoda — COMMENTED** [review]
I have reviewed the change and found some points, see my comments

**Comment by aryoda** [comment]
@buhtz Very good redaction, THX!

The "known issues" sub index is a great improvement for quick navigation!

What I love most: Italics for _Back In Time_ as proper name (I always wanted to apply exactly this format change some day!)

**Comment by buhtz** [comment]
> The "known issues" sub index is a great improvement for quick navigation!

I'm using https://derlin.github.io/bitdowntoc/ for that.

**Pull request merged**